src/internal_events/prometheus.rs

```diff
@@ -129,19 +129,13 @@ impl InternalEvent for PrometheusRemoteWriteSnapError {
 
 #[derive(Debug)]
 pub struct PrometheusRemoteWriteReceived {
-    pub byte_size: usize,
     pub count: usize,
 }
 
 impl InternalEvent for PrometheusRemoteWriteReceived {
     fn emit_logs(&self) {
         debug!(message = "Received remote_write events.", count = ?self.count);
     }
-
-    fn emit_metrics(&self) {
-        counter!("processed_events_total", self.count as u64);
```

Comment:
We could convert this to sink-specific metrics to preserve the data point

Reply:
We have the same metrics being emitted in `HttpSource` being used by this source.
https://github.com/timberio/vector/blob/ae18f8f7790c56bd916cc634a0cb818d1e6db9c4/src/sources/util/http.rs#L180-L183

Am I missing something here?